lib/buf.js

```diff
@@ -0,0 +1,30 @@
+const BACK_SLASH = 0x5C;
+
+class InputBuffer {
+    constructor(input) {
+        this._buf = input instanceof Buffer ? input : Buffer.from(input);
+        this._replace();
+    }
+
+    _replace() {
+        if (this._buf.indexOf(BACK_SLASH) < 0) {
```

Comment:
replace indexOf<0 with !includes

Reply:
I cannot understand the change here?
I understand, for the character, these two methods are the same.
As shown in this link:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#Polyfill

Reply:
You're right they are the same, but I think includes() is more readable since this is what you really want to achieve

Reply:
Yes, you are right. It's more semantic.
Changed.